cmake/ArborXConfig.cmake.in

```diff
@@ -16,7 +16,9 @@ if(Kokkos_ENABLE_HIP AND ARBORX_ENABLE_ROCTHRUST)
   find_dependency(rocthrust)
 endif()
 if(Kokkos_ENABLE_SYCL AND ARBORX_ENABLE_ONEDPL)
-  find_dependency(oneDPL)
+  if(!@ARBORX_COMPILER_HAS_ONEDPL_EXECUTION_HEADER@ OR !@ARBORX_COMPILER_HAS_ONEDPL_ALGORITHM_HEADER@)
```

Comment:
either define a temporary variable (something like `ARBORX_CXX_COMPILER_BUNDLED_ONEDPL`) or add a comment since this will be unreadable after the file is configured.